redbot/cogs/trivia/data/lists/worldcup.yaml

```
@@ -0,0 +1,432 @@
+- Rasunda Stadium
+- Råsunda Fotbollsstadion
+- Rasunda Fotbollsstadion
+- Råsunda Football Stadium
+- Rasunda Football Stadium
+- Råsunda
+- Rasunda
+"What was the name of the stadium where the 1954 World Cup final was held? (clue: the hosts were Switzerland)":
+- Wankdorfstadion
+"What was the name of the stadium where the 1950 World Cup final was held? (clue: the hosts were Brazil)":
+- Maracanã
+- Maracana
+- Maracanã Stadium
+- Maracana Stadium
+- Estádio Jornalista Mário Filho
+"What was the name of the stadium where the 1938 World Cup final was held? (clue: the hosts were France)":
+- Stade de Colombes
+- Stade Yves-du-Manoir
+- Stade olympique Yves-du-Manoir
+- Stade olympique de Colombes
+- Colombes
+"What was the name of the stadium where the 1934 World Cup final was held? (clue: the hosts were Italy)":
+- Stadio Nazionale PNF
+"What was the name of the stadium where the 1930 World Cup final was held? (clue: the hosts were Uruguay)":
+- Estadio Centenario
+
+# Continental confederations of international association football or NGBs
+
+The continental confederation "CONCACAF" governs sport in which continent so that their member nations can qualify for the world cup?:
+- North America
+The continental confederation "CONMEBOL" governs sport in which continent so that their member nations can qualify for the world cup?:
+- South America
+The continental confederation "UEFA" governs sport in which continent so that their member nations can qualify for the world cup?:
+- Europe
+The continental confederation "CAF" governs sport in which continent so that their member nations can qualify for the world cup?:
+- Africa
+The continental confederation "AFC" governs sport in which continent so that their member nations can qualify for the world cup?:
+- Asia
+The continental confederation "OFC" governs sport in which continent so that their member nations can qualify for the world cup?:
+- Australasia and Oceania
+- Australasia
+- Oceania
+
+# Number of teams
+
+From 2026 into the forseeable future, the tournaments will include how many teams?:
+- 48
+From 2002 to 2022, the tournaments included how many teams?:
+- 32
+From 1982 to 1994, the tournaments included how many teams?:
+- 24
+From 1954 to 1978, the tournaments included how many teams?:
+- 16
+How many teams were included in the 1950 World Cup hosted by Brazil?:
+- 13
+How many teams were included in the 1938 World Cup hosted by France?:
+- 15
+How many teams were included in the 1934 World Cup hosted by Italy?:
+- 16
+How many teams were included in the 1930 World Cup hosted by Uruguay?:
+- 13
+
+# General
+
+In which year was the World Cup founded?:
+- 1930
+By which organization is the World Cup governed by?:
+- FIFA
+Which country has the most World Cup titles?:
+- Brazil
+The World Cup has ran every four years since its inauguration, apart from the years ____ and ____ (due to World War Two).:
+- 1942 and 1946
+- 1946 and 1942
+- 1942, 1946
+- 1946, 1942
+- 1942 1946
+- 1946 1942
```

```suggestion
- "1942 1946"
- "1946 1942"
```
Just to be safe, in case it treats it like an `int` and tries to collapse away the space.

---

Yeah that's fair, better to be safe and certain. 👍